subsys/bluetooth/mesh/friend.c

```diff
@@ -1404,7 +1404,7 @@ static void friend_lpn_enqueue_rx(struct bt_mesh_friend *frnd,
 	 * this rx function. These packets have already been added to the
 	 * queue, and should be ignored.
 	 */
-	if (bt_mesh_elem_find(rx->ctx.addr)) {
+	if (bt_mesh_has_addr(rx->ctx.addr)) {
```

Comment:
If we modify this way, will not all messages sent to LPN(`sub list`)be discarded?

Reply:
Oh yes, I believe you're correct. In the instance where the message callback is set, any message which the LPN is subscribed too and the friend node is not subscribed too will be discarded. Nice catch.

I'll implement something to account for this.